src/glyphs/pr.svg

```diff
@@ -1,8 +1,4 @@
-<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
-<ellipse cx="3.66667" cy="13.0001" rx="1.66667" ry="1.66667" fill="#0C2739"/>
-<circle cx="12.6667" cy="13.0001" r="1.66667" fill="#0C2739"/>
-<ellipse cx="3.66667" cy="3.66679" rx="1.66667" ry="1.66667" fill="#0C2739"/>
-<line x1="3.83333" y1="4.66679" x2="3.83333" y2="13.3335" stroke="#0C2739"/>
-<path d="M12.6667 13.3335V7.33347C12.6667 5.12433 10.8758 3.33347 8.66667 3.33347L6.66667 3.33347" stroke="#0C2739"/>
-<path d="M8.66693 1.33333L6.66667 3.33359L8.66693 5.33384" stroke="#0C2739"/>
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Can you add screenshot?
This might change implementations since it's 24px instead of 16px

Reply:
@yhamry we need to ask Shuju and Emiliya to keep the icons in the same size